pkg/controllers/node/controller.go

```diff
@@ -59,7 +59,7 @@ type Controller struct {
 
 // Reconcile executes a reallocation control loop for the resource
 func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
-	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).Named("Node"))
+	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).Named("node").With("node", req.String()))
```

Comment:
With this change, I think we can change lines like 105 from
```
fmt.Errorf("patching node %s, %w", node.Name, err)
```
to
```
fmt.Errorf("patching node, %w", err)
```

Reply:
Done.